recipes/gloo/meta.yaml

```diff
@@ -0,0 +1,140 @@
+# To generate the version number
+{%- if false %}
+git clone git@github.com:facebookincubator/gloo.git
+cd gloo
+git pull
+echo 0.5.0.$(date  +%Y%m%d -d "$(git show -s --format=%ci HEAD)").$(git rev-list HEAD --count).$(git rev-parse --short HEAD)
```

Comment:
how do you choose 0.5.0?

Reply:
This is the version stated in the CMake files! https://github.com/facebookincubator/gloo/blob/main/CMakeLists.txt